tests/phpunit/tests/formatting/wpTrimExcerpt.php

```diff
@@ -92,4 +92,60 @@ public function test_should_generate_excerpt_for_empty_values() {
 		$this->assertSame( 'Post content', wp_trim_excerpt( null, $post ) );
 		$this->assertSame( 'Post content', wp_trim_excerpt( false, $post ) );
 	}
+
+	/**
+	 * Tests that `wp_trim_excerpt()` unhooks `wp_filter_content_tags()` from 'the_content' filter.
+	 *
+	 * @ticket 56588
+	 */
+	public function test_wp_trim_excerpt_unhooks_wp_filter_content_tags() {
+		$post = self::factory()->post->create();
+
+		/*
+		 * Record that during 'the_content' filter run by wp_trim_excerpt() the
+		 * wp_filter_content_tags() callback is not used.
+		 */
```

Comment:
Maybe overkill but assert it was originally present:

```suggestion
		 */
		 $this->assertSame( 10, has_filter( 'the_content', 'wp_filter_content_tags' ) );
```

Reply:
@westonruter I have been avoiding those kinds of assertions, to generally avoid multiple assertions in the same test, which is fine here since those situations are tested in another test, so we don't miss anything really.